includes/api/ApiQueryFilearchive.php

```diff
@@ -144,6 +144,18 @@ public function execute() {
 				$file['timestamp'] = wfTimestamp( TS_ISO_8601, $row->fa_timestamp );
 			}
 			if ( $fld_user ) {
+				/**
+				 * Check, how often is this code executed. Scope: the following if block.
+				 *
+				 * @author Mix
+				 * @see SUS-810
+				 */
+				if ( ( new Wikia\Util\Statistics\BernoulliTrial( 0.01 ) )->shouldSample() ) {
```

Comment:
Can we extract this to one method?
Maybe `PokazJakCzestoTuJestem( sampling , message)`?